config/supervisord.conf

```diff
@@ -38,7 +38,7 @@ command=/code/manage.py auto_settings
 environment=LOGGING_FILENAME=auto_settings_%(ENV_SERVER_NUMBER)s.log
 startretries=100
 startsecs=1
-autostart=false
+autostart=true
```

Comment:
ehhh, is tthis really required? Meaning: Why would we change that default and why in a hotfix?